test/API/CMakeLists.txt

```diff
@@ -332,93 +227,99 @@ if (HDF5_TEST_SERIAL)
       )
     endforeach ()
 
-    # Add tests for each external VOL connector that was built
```

Comment:
Everything here just shifted over once inside the `if (BUILD_SHARED_LIBS)` block